.vscode/setup.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+#Activate Python
+python -i
+
+#Install PIP
+python -m pip --version
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+
+#Install Poetry
+echo 'poetry --version' poetry --version
+
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+poetry install
```

Comment:
You can't run poetry install before cloning the repository. What list of packages do you expect this command to install?